### PR TITLE
bring django 2.0 compatibility

### PR DESCRIPTION
**Pull request by ticosax**
replaces #65

### CODE REVIEW

File: rules/templatetags/rules.py [hunk 1]
```diff
@@ -5,13 +5,19 @@
 
 register = template.Library()
 
+try:
+    assignment_tag = register.assignment_tag
+except AttributeError:
+    # django < 2.0
+    assignment_tag = register.simple_tag
```
Comment (dfunckt) [inline rules/templatetags/rules.py]: I think you got this the opposite way round? `register.assignment_tag` is Django < 2.0.

File: tests/testapp/urls.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 admin.autodiscover()
 
 urlpatterns = [
-    url(r'^admin/', include(admin.site.urls)),
+    url(r'^admin/', admin.site.urls),
```
Comment (dfunckt) [inline tests/testapp/urls.py]: Could you remove the `include` import as well?

File: tests/testsuite/contrib/test_admin.py [hunk 3]
```diff
@@ -1,5 +1,8 @@
-from django.core.urlresolvers import reverse
 from django.test import TestCase
+try:
+    from django.urls import reverse
+except ImportError:
+    from django.core.urlresolvers import reverse
```
Comment (dfunckt) [inline tests/testsuite/contrib/test_admin.py]: Do you mind adding a comment about which Django version this fallback is for?

File: tests/testsuite/contrib/test_views.py [hunk 4]
```diff
@@ -1,7 +1,10 @@
 from __future__ import absolute_import
 
 from django.core.exceptions import ImproperlyConfigured
-from django.core.urlresolvers import reverse
+try:
+    from django.urls import reverse
+except ImportError:
+    from django.core.urlresolvers import reverse
```
Comment (dfunckt) [inline tests/testsuite/contrib/test_views.py]: Same here.

File: .travis.yml [hunk 5]
```diff
@@ -1,61 +1,31 @@
...
+    - { env: TOXENV=py34-django17, python: 3.4 }
+    - { env: TOXENV=py34-django18, python: 3.4 }
+    - { env: TOXENV=py34-django19, python: 3.4 }
+    - { env: TOXENV=py34-django110, python: 3.4 }
+    - { env: TOXENV=py35-django18, python: 3.5 }
+    - { env: TOXENV=py35-django19, python: 3.5 }
+    - { env: TOXENV=py35-django110, python: 3.5 }
+    - { env: TOXENV=py34-django20, python: 3.4 }
+    - { env: TOXENV=py35-django20, python: 3.5 }
+    - { env: TOXENV=py36-django20, python: 3.6 }
```
Comment (dfunckt) [inline .travis.yml]: It seems you've missed combinations of PyPy, even though they're included in tox.ini

**Comment by rpkilby** [comment]
One consideration for Django 2.0 compat - the official recommendation is to only support Django 1.11+

File: .travis.yml (hunk 5 above)
Comment (ticosax) [reply .travis.yml]: sure I will re-add them

**Comment by ticosax** [comment]
@rpkilby This was my first attempt see #65

**Comment by ticosax** [comment]
I give up

**Comment by dfunckt** [comment]
@ticosax d2a88315 is based on this PR. Thanks!

**Pull request closed without merging**